### ISSUE TITLE
Strange behavior of Message object when used in async

### ISSUE THREAD

**Issue by MarcelBeining** [python-telegram-bot/python-telegram-bot#1311]
### Steps to reproduce
1. Add your bot to a group and to a channel
2. Run this python code (with your bot token)
```
import logging
import asyncio
from telegram.ext import (Updater,Filters,MessageHandler)

logging.basicConfig(level=logging.info)

_loop = asyncio.get_event_loop()

async def broadcastMsg(message):
    ''' 
    Function to broadcast a message to all linked channels.
    '''
    fromMedium = 'Telegram'
    logging.info(message)
    logging.info(message.from_user)
    
    if message.from_user is not None:
        author = message.from_user.username
    else:
        author = None
    print(message.from_user.username)  # running this miraculously stops execution at this line without an error
    channel = message.chat.title
    content = message.text
    logging.info('Message received from %s on %s: %s'%(author,channel,content))
    
    
def on_TMessage(bot,update):
    # check if this chat is already known, else save it
    if update.message:
        asyncio.run_coroutine_threadsafe(broadcastMsg(update.message), _loop)
    elif update.channel_post:
        asyncio.run_coroutine_threadsafe(broadcastMsg(update.channel_post), _loop)
        
        
messageHandler = MessageHandler(Filters.text, on_TMessage)
updaterTelegram = Updater(token = XXXXXXXX, request_kwargs={'read_timeout': 10, 'connect_timeout': 10})
updaterTelegram.dispatcher.add_handler(messageHandler)
updaterTelegram.start_polling()

```
3. Write a message in the group
4. Write a message in the channel

### Expected behaviour
For the channel, as there is no "from_user", it should simply output an Python attribute error like 

> AttributeError: 'NoneType' object has no attribute 'username'

### Actual behaviour
For the group, everythin works as expected, but for the channel nothing happens, the code is not executed further but also no Error pops up, even if I surround the line with a try/catch block
This is what the log says when writing to the channel (chat id removed):

> 2018-12-11 17:42:35,154 [MainThread  ] [INFO ]  {'message_id': 110, 'date': 1544546555, 'chat': {'id': -XXXXXXXX, 'type': 'channel', 'title': 'ThisIsATestChannel'}, 'text': 'test32', 'entities': [], 'caption_entities': [], 'photo': [], 'new_chat_members': [], 'new_chat_photo': [], 'delete_chat_photo': False, 'group_chat_created': False, 'supergroup_chat_created': False, 'channel_chat_created': False}
> 2018-12-11 17:42:35,155 [MainThread  ] [INFO ]  None

**I think this issue could be related to that issue:**
https://github.com/python-telegram-bot/python-telegram-bot/issues/210
**I need to use that async routine because my program has to cooperate with another library that only runs in async**

### Configuration
**Operating System:**
Windows 10 64bit

**Version of Python, python-telegram-bot & dependencies:**
3.6
11.1.0

### Logs
Insert logs here (if necessary)

**Comment by jsmnbom** [comment]
Is there still no output if you enable debugging logs?

**Comment by MarcelBeining** [comment]
> Is there still no output if you enable debugging logs?

No output at least that helps, see here (starting with the second I have written to the telegram channel; tokens removed):

> 2018-12-12 14:06:47,426 [updater     ] [DEBUG]  https://api.telegram.org:443 "POST /XXXXXX/getUpdates HTTP/1.1" 200 190
> 2018-12-12 14:06:47,427 [updater     ] [DEBUG]  Getting updates: [497425035]
> 2018-12-12 14:06:47,428 [updater     ] [DEBUG]  [<telegram.update.Update object at 0x000001CBA119DD68>]
> 2018-12-12 14:06:47,429 [updater     ] [DEBUG]  Exiting: get_updates
> 2018-12-12 14:06:47,429 [updater     ] [DEBUG]  Entering: get_updates
> 2018-12-12 14:06:47,429 [dispatcher  ] [DEBUG]  Processing Update: {'update_id': 497425035, 'channel_post': {'message_id': 112, 'date': 1544620006, 'chat': {'id': -1001139634443, 'type': 'channel', 'title': 'ThisIsATestChannel'}, 'text': 'test34', 'entities': [], 'caption_entities': [], 'photo': [], 'new_chat_members': [], 'new_chat_photo': [], 'delete_chat_photo': False, 'group_chat_created': False, 'supergroup_chat_created': False, 'channel_chat_created': False}}
> 2018-12-12 14:06:47,432 [MainThread  ] [INFO ]  {'message_id': 112, 'date': 1544620006, 'chat': {'id': -1001139634443, 'type': 'channel', 'title': 'ThisIsATestChannel'}, 'text': 'test34', 'entities': [], 'caption_entities': [], 'photo': [], 'new_chat_members': [], 'new_chat_photo': [], 'delete_chat_photo': False, 'group_chat_created': False, 'supergroup_chat_created': False, 'channel_chat_created': False}
> 2018-12-12 14:06:47,434 [MainThread  ] [INFO ]  None
> 2018-12-12 14:06:57,431 [updater     ] [DEBUG]  Timed out getting Updates: Timed out
> 2018-12-12 14:06:57,434 [updater     ] [DEBUG]  Entering: get_updates

**Comment by jsmnbom** [comment]
Hmm, well in that case this does seem like a bug _of some kind_ as far as I can tell.
We don't really officially support asyncio (simply by virtue of us never having tested it), so I can't really promise when we'll get around to fixing anything though.

**Comment by Eldinnie** [comment]
Hi @MarcelBeining just to be sure. Try adding an error handler and ty again.

**Comment by MarcelBeining** [comment]
Hi @Eldinnie .
As I wrote in my initial post
> the code is not executed further but also no Error pops up, even if I surround the line with a try/catch block

;-)

**Comment by Eldinnie** [comment]
@MarcelBeining Which is not the same as adding an errorhandler to the bot with `dispatcher.add\_error\_handler`. So could you humor me and try that?

**Comment by MarcelBeining** [comment]
Hmmm, I added an error handler as you suggested but that did not help.
However I retried to add a try/catch block to the critical code with logging of the error and now it shows the error correctly! 
Maybe I had some errorneous try/catch block before so that the catch failed, too, and I couldn't see as errors in coroutines are not shown in the console. =/

So actually I can close the issue now, and say sorry for taking your precious time!
